prusti-viper/src/encoder/middle/core_proof/builtin_methods/interface.rs

```diff
@@ -894,7 +919,7 @@ impl<'p, 'v: 'p, 'tcx: 'v> BuiltinMethodsInterface for Lowerer<'p, 'v, 'tcx> {
                 vir_mid::TypeDecl::TypeVar(_) => {
                     // move_place of a generic variable has no body
                 }
-                vir_mid::TypeDecl::Tuple(decl) => {
+                vir_mid::TypeDecl::Tuple(ref decl) => {
```

Comment:
Instead of adding `ref` to every branch, you could change `match type_decl` to `match &type_decl`.
```suggestion
                vir_mid::TypeDecl::Tuple(decl) => {
```

Reply:
done